_components/tooltip/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/tooltip/accessibility-tests/
+layout: accessibility-test
+component:
+  name: tooltip
+title: Tooltip accessibility tests
+category: Components
+lead: Any USWDS tooltip should pass these manual accessibility tests.
```

Comment:
Adding component here for consistency 
```suggestion
lead: Any USWDS tooltip component should pass these manual accessibility tests.
```

Reply:
Good catch. Added in [1ced497](https://github.com/uswds/uswds-site/pull/2646/commits/1ced49732cf1854a229de37d3eb993fa95c8c726).